src/content/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model.mdx

```diff
@@ -11,117 +11,74 @@ This tutorial will walk you through some common procedures for managing users on
+* Assigning new group access grants requires Pro or Enterprise tier. For details, see [user management requirements](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#requirements).
 * [Role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) requirements:
-  * Steps 1-4: **Authentication domain manager** and **Organization manager** roles.
-  * Step 5: **Authentication domain manager** role.
+  * Using the **Organization and access** UI to manage group access: requires **Authentication domain manager** and **Organization manager** roles.
+  * Using the **User management** UI to add/edit users: requires **Authentication domain manager** role.
 
 ## Overview
 
-This tutorial uses terms and concepts explained in these introductory docs:
+Before using this tutorial, we first recommend reading these docs:
 
-* [New Relic One user model basics](/docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-one-user-model)
-* [User management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles)
+* [Where to find user management features in the UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
+* [Important user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
 
 This tutorial will walk you through how to:
 
-* Step 1: [Add/manage groups](#step-1-groups)
-* Step 2: [Add/manage roles](#step-2-roles)
-* Step 3: [Manage accounts](#step-3-roles)
-* Step 4: [Grant groups access to roles/accounts](#step-4-access-grants) (access grants)
-* Step 5: [Add/manage users](#step-5-users)
+* [View available accounts](#accounts)
+* [How to grant groups access to roles and accounts](#group-access) 
+* [How to create custom roles](#roles)
+* [How to add users to groups](#add-users)
 
-<Callout variant="tip">
-  The tutorial steps are also summarized in the [Organization and access UI](https://one.newrelic.com/launcher/organizations-ui.organization-launcher).
-</Callout>
-
-## Step 1: Add/manage groups [#step-1-groups]
+## View available accounts [#accounts]
 
-Groups are used to manage what users are able to do in New Relic. By default, most New Relic organizations have two groups: [**Admin** and **User**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#group-roles). And you can add your own custom groups.
-
-<Callout variant="tip">
-  You can also [import users automatically from your identity provider](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign).
-</Callout>
+Understanding the accounts in your New Relic organization can be an important first step because when you grant groups access, you must choose what account to give the group access to.
```

Comment:
Could this say, "Understanding the accounts in your New Relic organization is important for granting groups access, because you must choose what accounts to grant them access to."